OpenRA.Mods.Common/Traits/Buildings/Building.cs

```diff
@@ -136,6 +143,12 @@ public IEnumerable<CPos> PathableTiles(CPos location)
 				yield return t;
 		}
 
+		public IEnumerable<CPos> HotTiles(CPos location)
```

Comment:
I'm not sure that the meaning of a "hot" tile is going to be clear to anybody who doesn't already understand what is using this. Can we please rename this to something like `PathableNoStoppingTiles`?

Reply:
Renamed.